pkg/login/social/connectors/azuread_oauth.go

```diff
@@ -75,7 +75,7 @@ type keySetJWKS struct {
 func NewAzureADProvider(info *social.OAuthInfo, cfg *setting.Cfg, ssoSettings ssosettings.Service, features featuremgmt.FeatureToggles, cache remotecache.CacheStorage) *SocialAzureAD {
 	config := createOAuthConfig(info, cfg, social.AzureADProviderName)
```

Comment:
Not really related to this pr but all providers calls `createOAuthConfig` and then just pass config to `newSocialBase`. We already pass all required information to `newSocialBase` to construct the config there right?

Reply:
No we pass that to `newSocialBase` as first argument?

Reply:
😅

Reply:
Done